apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.types.ts

```diff
@@ -15,6 +16,10 @@ export const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
   assessment_question: AssessmentQuestionSchema,
   user_or_group_name: z.string().nullable(),
   open_issue_count: z.number().nullable(),
+  ai_grading_status: z.enum(['None', 'Graded', 'Outdated', 'Latest']),
+  last_human_grader: z.string().nullable(), // null if not graded by human
+  rubric_difference: RubricItemSchema.extend({ false_positive: z.boolean() }).array().nullable(),
```

Comment:
AFAICT almost all of the data returned by this is unused. Given the potentially large list of instance questions, I'd strongly consider picking off just the properties that we actually need in the query. cc @mwest1066, I'm hoping this is the kind of situation where you're comfortable not fetching entire row objects, but let me know if that's not the case.

Reply:
Yes, absolutely. Although I would prefer that this type had an instance question as a member, rather than extending instance question. In general I think the type-extension pattern is bad.